lib/routes.js

```diff
@@ -103,6 +103,8 @@ router
     switch (ctx.params.queryMethod) {
       case "delete":
         return objectCtrl.deleteMultipleObjects(ctx);
+      case undefined:
+        return objectCtrl.postObject(ctx);
```

Comment:
Could you make this consistent with the other route handlers and keep the `undefined` case at the top of the switch statement?

Reply:
done